.env.example

```diff
@@ -1,6 +1,14 @@
 # general
 NEXT_PUBLIC_API_URL=http://localhost:3000
-DATABASE_URL=mysql://root:@localhost:3306/shibaac
+
+MYSQL_PORT=3306
+MYSQL_ROOT=root
+MYSQL_DATABASE=test_db
+MYSQL_PASSWORD=secret
+MYSQL_ROOT_PASSWORD=secret
+MYSQL_HOST=localhost
+
+DATABASE_URL="mysql://root:secret@localhost:3306/test_db"
```

Comment:
is this duplicated? or why is there two database credentials types

Reply:
no, it's actually the credentials for docker.